From 12fbfa26a432e551934f0b434b77ad92b995324a Mon Sep 17 00:00:00 2001
From: David Disseldorp <ddiss@suse.de>
Date: Tue, 31 Oct 2023 11:36:48 +0100
Subject: [PATCH] ksmbd-tools: fix worker_init_sa_handler return type

The function doesn't return any errors, so switch it to void to silence
compiler warnings such as:
  mountd.c: In function 'worker_init_sa_handler':
  mountd.c:114:1: error: no return statement in function returning
		  non-void [-Werror=return-type]
    114 | }
        | ^

Signed-off-by: David Disseldorp <ddiss@suse.de>
---
 mountd/mountd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mountd/mountd.c b/mountd/mountd.c
index 0505698..8f2af7d 100644
--- a/mountd/mountd.c
+++ b/mountd/mountd.c
@@ -97,7 +97,7 @@ static void worker_sa_sigaction(int signo, siginfo_t *siginfo, void *ucontext)
 	_Exit(128 + signo);
 }
 
-static int worker_init_sa_handler(sigset_t sigset)
+static void worker_init_sa_handler(sigset_t sigset)
 {
 	int signo;
 
-- 
2.35.3

